src/pages/product-list-page/products-per-page/products-per-page.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { Button, ButtonGroup } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import styles from './products-per-page.styles';
+import { setProductsPerPage } from '../../../redux/products/products.actions';
+import { ITEMS_PER_PAGE } from '../../../translations/product-list.translations';
+
+const productsCount = sessionStorage.getItem('productsPerPage') || 9;
+
+const ProductsPerPage = () => {
```

Comment:
Please, rename component and its files to the `CountPerPage`, it is much more explicit that it controls count

Reply:
fixed